app/controllers/application_controller.rb

```diff
@@ -274,15 +274,19 @@ def set_locale
     I18n.locale = LocaleChooser.new(params[:locale], request).locale
   end
 
+  def sp_session_ial
+    sp_session[:ial2] ? 2 : 1
+  end
```

Comment:
What if we updated `StoreSpMetadaInSession` to store the literal IAL level? `ial: ial_context.ial`. Then we could get rid of this ternary? Because of the way this is set, if an SP requests IAL2 strict, this would be false and would return `1`
 
Then it would give us the values like `22` for `IAL2_STRICT` or `0` for `IAL_MAX` in case we ever need to differentiate?

This part: https://github.com/18F/identity-idp/blob/master/app/services/store_sp_metadata_in_session.rb#L35-L47

Reply:
Thanks for raising the issue with IAL2 strict! I made the suggested changes.